blockchain_connector/avalon_blockchain_connector/ethereum/ethereum_connector.py

```diff
@@ -41,7 +42,7 @@
 logging.basicConfig(
     format="%(asctime)s - %(levelname)s - %(message)s", level=logging.INFO)
 
-LISTENER_SLEEP_DURATION = 5  # seconds
+WORKER_SYNCUP_SLEEP_DURATION = 5  # seconds
```

Comment:
Why do we need this sleep duration? Moment we start listener it will have workers available in lmdb because enclave adds to that worker entry. Add a dependency in docker to start connector after enclave manager and listener.

Reply:
There's [dependency in docker-compose](https://github.com/hyperledger/avalon/blob/master/docker-compose-eth-besu.yaml#L140) already. But, I saw this happen a couple of times when I was repetitively doing docker-compose down & up. Hence this change.